code/game/objects/items/weapons/storage/belt.dm

```diff
@@ -33,7 +33,19 @@
 		/obj/item/device/analyzer,
 		/obj/item/weapon/extinguisher/mini,
 		/obj/item/device/radio,
-		/obj/item/clothing/gloves
+		/obj/item/clothing/gloves,
+		/obj/item/device/geiger_counter,
+		/obj/item/weapon/tank/internals/emergency_oxygen,

```

Comment:
Wasn't there a consistency issue with tanks on the belt not working in internals?